resharper/resharper-unity/test/data/CSharp/Daemon/Stages/Analysis/PropertyDrawerOnGUIBaseCallWarning.cs.gold

```diff
@@ -0,0 +1,32 @@
+﻿using UnityEditor;
+using UnityEngine;
+
+public class Test : PropertyDrawer
+{
+    public override void OnGUI(Rect position, SerializedProperty property, GUIContent label)
+    {
+        var a = new Test();
+        a.OnGUI(position, property, label);
+        base.|OnGUI|(0)(position, property, label);
+        
+        OnGUI();
+        OnGUI(position, property, label);
+    }
+    
+    public void OnGUI()
+    {
+        var a = new SerializedObject(new Object());
+        var b = a.FindProperty("test");
+        base.OnGUI(new Rect(), b, new GUIContent("test"));
+    }
+}
+
+public class PlainClass
+{
+    public void OnGUI(Rect position, SerializedProperty property, GUIContent label)
+    {
+        OnGUI(position, property, label);
```

Comment:
I think this will just be a recursive call. How about adding a base class, so the test does call `base.OnGUI`, but it's not `PropertyDrawer.OnGUI`, so doesn't highlight?